modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/StringUtils.java

```diff
@@ -92,6 +89,18 @@ public static String underscore(final String word) {
         });
     }
 
+    private static String escapeWordTokens(String result) {
```

Comment:
maybe a comment here with an example DBaaS - to Dbaas?

Reply:
done ✔️ thank you!